src/maptm/src/maptm_eligibility.c

```diff
@@ -518,13 +509,12 @@ void maptm_eligibilityStart(int WanConfig)
 {
     init_eligibility();
 
-    // If Cloud did not set MAP-T tables, set default MAP-T support value
-    if (!maptm_ovsdb_tables_ready()) 
-    {
-        WanConfig |= MAPTM_ELIGIBILITY_ENABLE;   
-        maptm_dhcp_option_update_15_option(strucWanConfig.mapt_support);
-        maptm_dhcp_option_update_95_option(strucWanConfig.mapt_support);
-    }
+    // Check mapt_support value and update WanConfig and requested options list
+    LOGT("%s mapt_support is %s" , __func__, strucWanConfig.mapt_support ? "enabled" : "disabled");
+    if (strucWanConfig.mapt_support) WanConfig |= MAPTM_ELIGIBILITY_ENABLE;
+    else WanConfig &= MAPTM_IPV6_ENABLE;
```

Comment:
```suggestion
    if (strucWanConfig.mapt_support) {
        WanConfig |= MAPTM_ELIGIBILITY_ENABLE;
    } else {
        WanConfig &= MAPTM_IPV6_ENABLE;
    }
```

Reply:
Hi Lars,
In the code style document, it is mentioned that we don't need to add brackets for one line conditional statements. Doesn't that exception apply to this case?

Reply:
Hi @MalekSST,

Omission of braces is allowed for one-liners, but in this case you have two lines.

From the Coding Style:

**Example ("if ... else" one-liner):**
```C
for (i=0; i<QCSAPI_MAX_ETHER; i++) {
    if (hide_mac) mac_addr[i] = 0;                  /* WRONG */
    else          mac_addr[i] = (uint8_t)tmp[i];    /* WRONG */
}

/* either expand it ... */
for (i=0; i<QCSAPI_MAX_ETHER; i++)
{
    if (hide_mac) {
        mac_addr[i] = 0;
    } else {
        mac_addr[i] = (uint8_t)tmp[i];
    }
}

/* ... or use a one-liner */
for (i=0; i<QCSAPI_MAX_ETHER; i++) {
    if (hide_mac) { mac_addr[i] = 0; } else { mac_addr[i] = (uint8_t)tmp[i]; }
}
```

> Specifically for the "if...else" statement, if it is written as a one-liner, it is advisable to use braces
to make the two clauses stand out.
However, only do this if the intention is easy to understand and there is no need to add a
comment for each of the actions. In general code, it is usually better to avoid "if...else"
one-liners, but they may be useful in macros.